package.json

```diff
@@ -31,11 +31,10 @@
         ".": {
             "import": "./dist/prefect-ui-library.mjs",
             "require": "./dist/prefect-ui-library.umd.js"
-        },
-        "./dist/style.css": "./dist/style.css"
```

Comment:
Downstream applications will need to update references to this file (now available via `src/styles`

Reply:
This doesn't make sense to me. The css is still part of the build and generally needs to be designated as an export in the package.json or things like ts will error when trying to import it. At least that's how I understand it.

Reply:
The change to the `files` entry will address this by making both the source code available (via `/src/styles`) and the build artifacts available (via `/dist`). Retaining this mapping doesn't make a lot of sense to me because all it was saying was you could find a file at `/dist/style.css`, which is still true but there are a lot of other files there too. The import/require mappings are still correct because they're signals to node about how to handle the different import styles.